AISKU/scripts/listCdnVersions.ps1

```diff
@@ -4,6 +4,8 @@ param (
     [string] $sasToken = $null,                         # The SAS Token to use rather than using or attempting to login
     [string] $logPath = $null,                          # The location where logs should be written
     [switch] $showFiles = $false,                       # Show the individual files with details as well
+    [switch] $inclExt = $false,                         # Include the extensions
+    [switch] $activeOnly = $false,                      # Only show the active (deployed) versions
```

Comment:
This will restrict the listing to just the major/minor versions (v2; v2.x) and not the individual base versions.